gnubuilder.Dockerfile.template

```diff
@@ -0,0 +1,12 @@
+FROM rust:%%RUST_VERSION%% AS builder
```

Comment:
Builds unFTP binary with dynamically linked libc and pam.so and the outputs to stdout for us to write to the host filesystem.